syntax = "proto3";
package QikkDB.NetworkClient.Message;

message SetDatabaseMessage {
    string DatabaseName = 1;
}